Assets/tab1.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System.Collections;
```

Comment:
I think you should just write a single `Tab` class that registers to the `TabManager` on `Awake()`